reckoner/helm/client.py

```diff
@@ -172,7 +179,12 @@ def _find_version(raw_version):
         if ver:
             return ver.group(1)
         else:
-            return None
+            logging.debug("Couldn't find Helm version. Are we using Helm 3?")
+            try:
+                ver = re.compile(r'v([0-9\.]+)(\+[a-zA-Z]+)(\+g[0-9a-f]+)?').search(str(raw_version))
```

Comment:
Having compiles within a method can be inefficient for memory since you're continually initializing it.

Note this is not a problem for performance but just a heads up as to why the original implementation is a class variable

Example of performance:
```python
import re, timeit

my_regexp = re.compile(r'a')

def outside_of_func():
        my_regexp.search(str("asdf"))

def inside_of_func():
        r = re.compile(r'a').search(str("asdf"))

if __name__ == '__main__':
        print("inside")
        print(timeit.timeit(lambda: inside_of_func(), number=100000))
        print("outside")
        print(timeit.timeit(lambda: outside_of_func(), number=100000))
```